third_party/move/move-vm/integration-tests/src/tests/binary_format_version.rs

```diff
@@ -3,88 +3,96 @@
 
 use move_binary_format::{
     deserializer::DeserializerConfig,
-    file_format::{basic_test_module, basic_test_script},
+    file_format::basic_test_script,
     file_format_common::{IDENTIFIER_SIZE_MAX, VERSION_MAX},
 };
 use move_core_types::{account_address::AccountAddress, vm_status::StatusCode};
-use move_vm_runtime::{config::VMConfig, module_traversal::*, move_vm::MoveVM, DummyCodeStorage};
+use move_vm_runtime::{
+    config::VMConfig, module_traversal::*, move_vm::MoveVM, IntoUnsyncCodeStorage,
+    LocalModuleBytesStorage,
+};
 use move_vm_test_utils::InMemoryStorage;
 use move_vm_types::gas::UnmeteredGasMeter;
 
 #[test]
 fn test_publish_module_with_custom_max_binary_format_version() {
-    let m = basic_test_module();
-    let mut b_new = vec![];
-    let mut b_old = vec![];
-    m.serialize_for_version(Some(VERSION_MAX), &mut b_new)
-        .unwrap();
-    m.serialize_for_version(Some(VERSION_MAX.checked_sub(1).unwrap()), &mut b_old)
-        .unwrap();
+    // TODO(loader_v2): Reimplement this test in Aptos codebase to check for deserialization.
```

Comment:
I guess since we are on the branch it's fine to just disable the test for now. the rationale is that publish logic will leave elsewhere, right?

Reply:
Yes, it was just a temporary thing, brought back here: #14395